Cargo.toml

```diff
@@ -13,21 +13,18 @@ doctest = false
 [features]
 unstable = [
     "string_cache/unstable",
-    "tendril/unstable",
     "rc/unstable",
     "html5ever/unstable",
     "selectors/unstable",
 ]
-with-hyper = ["hyper"]
```

Comment:
Servo does uses html5ever and hyper, but probably won’t use this method: it has more complex code for networking and document loading anyway.

I don’t know if this is a good idea, I’m making things up as I go along. I moved it because it seemed easier to have kuchiki’s `parse_html` return `html5ever::Parser` (which in turn has a method that returns `html5ever::BytesParser) rather than have than have wrapper types that do little else than re-implement every method and trait to forward calls. At the same time, I prefer inherent methods over extension traits since users need to import traits separately. But perhaps we should advise users (through docs) to do something like `use kuchiki::traits::*;` instead.

(The point of these return types is that they implement the `TendrilSink` trait to allow users to feed input incrementally: https://github.com/servo/tendril/blob/master/src/stream.rs)

What do you think?